infra/aws/terraform/s3/k8s-infra-e2e-boskos-scale-001/.gitignore

```diff
@@ -0,0 +1,21 @@
+# Local .terraform directories
+**/.terraform/*
+
```

Comment:
@pkprzekwas suggestion: we should probably move this .gitignore file to `../infra/aws/terraform` so all the sub-repos inherit this config.

Reply:
We can think about having some generic ignore at the top level of the terraform directory. But that should be a dedicated PR that would also remove other ignores.